test/scope.test.js

```diff
@@ -76,6 +76,24 @@ describe('scope', function () {
     });
   });
   
+  it('should not cache any results', function (done) {
+    Station.scope('active', {where: {isActive: true}});
+    Station.active.create(function (err, s) {
+      s.isActive.should.be.true;
```

Comment:
Please handle the error case:

```
if (err) return done(err);
```

Without that, the error details are lost and the test fails with rather unhelpful error "TypeError: Cannot read property 'isActive' of undefined".